src/ImageSharp.Formats.Png/PngEncoderCore.cs

```diff
@@ -495,7 +495,7 @@ private QuantizedImage<TColor> WritePaletteChunk<TColor>(Stream stream, PngHeade
 
             if (this.Quantizer == null)
             {
-                this.Quantizer = new WuQuantizer<TColor>();
+                this.Quantizer = new OctreeQuantizer<TColor>();
```

Comment:
I just wondered why did you change PngDecoder in the Dithering PR :)
But at least it helped noticed this change :)
I made this benchmark a month ago:
```
                               Method | InputCategory |          Mean |   Scaled | Scaled-StdDev | Allocated |
------------------------------------- |-------------- |-------------- |--------- |-------------- |---------- |
     'EncodeGifMultiple - ImageSharp' |     AllImages | 8,283.9980 ms | 3,691.26 |          0.00 |  72.11 MB |
 'EncodeGifMultiple - System.Drawing' |     AllImages |     2.2442 ms |     1.00 |          0.00 |   3.22 MB |
```
Ok, it's running on multiple Gif frames, while in PNG we have only one, but still very strange.